packages/components/nodes/sequentialagents/commonUtils.ts

```diff
@@ -99,7 +99,22 @@ export const transformObjectPropertyToFunction = (obj: ICommonObject, state: ISe
                     (message) => message.additional_kwargs && message.additional_kwargs?.nodeId === parsedValue.id
                 )
                 const messageOutput = messageOutputs[messageOutputs.length - 1]
-                if (messageOutput) value = messageOutput.content
+                if (messageOutput) {
+                    // if messageOutput.content is a string, set value to the content
+                    if (typeof messageOutput.content === 'string') value = messageOutput.content
+                    // if messageOutput.content is an array
+                    else if (Array.isArray(messageOutput.content)) {
```

Comment:
should we also check if the array is not empty?

Reply:
yeah that makes sense... standby

Reply:
Hmm... should we throw an error if it's empty or...?

Reply:
that's what i added in there